  INFO root [2019-03-07 13:42:08,646] - 
  INFO root [2019-03-07 13:42:08,646] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-07 13:42:08,663] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-07 13:42:08,663] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/skiing3_server_assets/;D:/kbengine-2.4.2/skiing3_server_assets/scripts/;D:/kbengine-2.4.2/skiing3_server_assets/res/
  INFO root [2019-03-07 13:42:08,664] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-07 13:42:08,686] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:7773 )
  INFO root [2019-03-07 13:42:08,687] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:7773, SOMAXCONN=5.
  INFO root [2019-03-07 13:42:08,687] - server-configs:
  INFO root [2019-03-07 13:42:08,687] - 	internalTcpAddr : 192.168.0.148:7773
  INFO root [2019-03-07 13:42:08,687] - 	componentID : 5000
  WARN root [2019-03-07 13:42:08,700] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-07 13:42:08,796] - Components::findLogger: found logger, addr:192.168.0.148:7760
  INFO root [2019-03-07 13:42:08,805] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-07 13:42:08,805] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-07 13:42:08,816] - ---- baseappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: E6918B69E98EAAAB4657C1417BA2685B. Config: Debug X86. Built: 17:20:25 Feb 27 2019. AppID: 5000. UID: 28936. PID: 5836 ----
  INFO root [2019-03-07 13:42:08,817] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-07 13:42:08,825] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-07 13:42:08,826] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-07 13:42:09,200] - ---- baseappmgr is running ----
 DEBUG root [2019-03-07 13:42:09,201] - Components::process(): Request for the process of identity...
  WARN root [2019-03-07 13:42:15,313] - Forwarding logs to logger[192.168.0.148:7760/0/0/0]...
 ERROR root [2019-03-07 13:42:57,520] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-07 13:42:57,521] - The following logs sent to logger failed:
  WARN root [2019-03-07 13:42:57,521] - ==> [2019-03-07 13:42:57 520] TCPPacketReceiver::processPendingEvents(192.168.0.148:7760): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-07 13:42:57,522] - ==> [2019-03-07 13:42:57 520] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-07 13:42:58,152] - TCPPacketReceiver::processPendingEvents(192.168.0.148:7763): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-07 13:42:58,153] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-07 13:42:58,163] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-07 13:42:58,168] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-07 13:42:58,169] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-03-07 13:42:58,211] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-03-07 13:42:58,528] - ThreadPool::destroy(): waiting for thread(2)[0x1dd71c8,0x1dd7508,], try=1
